<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>186</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QLabel" name="background_label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>292</width>
     <height>155</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>player_wall.jpg</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="fixed">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>7</y>
     <width>17</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="mini_top">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>7</y>
     <width>17</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>gg.png</normaloff>gg.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="min">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>7</y>
     <width>17</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>gg.png</normaloff>gg.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>7</y>
     <width>17</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>gg.png</normaloff>gg.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_play">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="preview">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>136</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="next">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>136</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="music_name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lrc_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="music_list">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>145</y>
     <width>31</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_lrc">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>145</y>
     <width>31</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="state_label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_pause">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="song_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
